### DISCUSSION TITLE
Sidebar Collapsed event

### DISCUSSION THREAD

**Discussion by Lasoty** [Megabit/Blazorise#3628] [Q&A]
Hi, 
is there any way to hook an event in which Sidebar is collapsed or expanded?

**Comment by stsrki** [comment]
Do you mean Sidebar extension or Bar component? If it's the Sidebar then no, it doesn't have any event that gives you collapse information. We're not actively working on it because all the work is now done on the newest Bar component. Which also have much more features, including the `VisibleChanged` event.
